js/content/community.js

```diff
@@ -1523,6 +1523,8 @@ let StatsPageClass = (function(){
             let node = item[1];
             personal.insertAdjacentElement("beforeend", node);
         }
+
+        SyncedStorage.set("sortachievementsby", key);
```

Comment:
Stuff like that is normally saved in `LocalStorage`

Reply:
All other existing sorting functions also used `SyncedStorage`, I was just following their example.

Reply:
I just had this example of the HD playback option in my mind.
In the end I think it doesn't really matter, `SyncedStorage` is more persistent, but `LocalStorage` is more suitable for such information imo.
We should only use one though